<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QMainWindow" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreditCalc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(42, 42, 42);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Eq_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:none;
	border-radius:15px;
	background-color:rgb(113, 43, 117);
	color:rgb(255,255,255);
	min-height:40px;
	max-height:40px;
	min-width:130px;
	max-width:130px;
}
QPushButton:hover
{
	border:none;
	border-radius:15px;
	background-color:rgb(164, 103,138);
	color:rgb(255,255,255);
	min-height:40px;
	max-height:40px;
	min-width:130px;
	max-width:130px;
}
QPushButton:pressed
{
	border:none;
	border-radius:15px;
	background-color:rgb(0, 0,0);
	color:rgb(255,255,255);
	min-height:40px;
	max-height:40px;
	min-width:130px;
	max-width:130px;
}</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>497</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Amount_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border:none;
	border-radius:25px;
	background-color:rgb(255, 159, 28);
	color:rgb(255,255,255);
}</string>
       </property>
       <property name="text">
        <string>Сумма кредита</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QSpinBox" name="Amount_spinBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QSpinBox {
color: white;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>100000000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>500000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Term_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border:none;
	border-radius:25px;
	background-color:rgb(255, 159, 28);
	color:rgb(255,255,255);
}</string>
       </property>
       <property name="text">
        <string>Срок кредита</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QSpinBox" name="Term_spinBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QSpinBox {
color: white;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QRadioButton" name="Years_button">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
color: white;
}</string>
       </property>
       <property name="text">
        <string>лет</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QRadioButton" name="Months_button">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
color: white;
}</string>
       </property>
       <property name="text">
        <string>месяцев</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Rate_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border:none;
	border-radius:25px;
	background-color:rgb(255, 159, 28);
	color:rgb(255,255,255);
}</string>
       </property>
       <property name="text">
        <string>Процентная ставка</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="Rate_doubleSpinBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QDoubleSpinBox {
color: white;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Type_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border:none;
	border-radius:25px;
	background-color:rgb(255, 159, 28);
	color:rgb(255,255,255);
}</string>
       </property>
       <property name="text">
        <string>Тип ежемесячных платежей</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="Ann_button">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
color: white;
}</string>
       </property>
       <property name="text">
        <string>Аннуитетные</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="2" colspan="3">
      <widget class="QRadioButton" name="Diff_button">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
color: white;
}</string>
       </property>
       <property name="text">
        <string>Дифференцированные</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Month_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border:none;
	border-radius:25px;
	background-color:rgb(255, 159, 28);
	color:rgb(255,255,255);
}</string>
       </property>
       <property name="text">
        <string>Месяц:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="Month_spinBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QSpinBox {
color: white;
}</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>501</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="Payment_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border:none;
	border-radius:25px;
	background-color:rgb(199, 75, 80);
	color:rgb(255,255,255);
}</string>
       </property>
       <property name="text">
        <string>Ежемесячный платеж:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Payment">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border:none;
	background-color:rgb(255, 255,255);
	color:rgb(0,0,0);
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Percentage_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border:none;
	border-radius:25px;
	background-color:rgb(199, 75, 80);
	color:rgb(255,255,255);
}</string>
       </property>
       <property name="text">
        <string>Начисленные проценты:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="Percentage">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border:none;
	background-color:rgb(255, 255,255);
	color:rgb(0,0,0);
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="All_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border:none;
	border-radius:25px;
	background-color:rgb(199, 75, 80);
	color:rgb(255,255,255);
}</string>
       </property>
       <property name="text">
        <string>Долг+проценты:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="All">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border:none;
	background-color:rgb(255, 255,255);
	color:rgb(0,0,0);
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
